pkg/api/v1/deepcopy_generated.go

```diff
@@ -1,7 +1,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2018 The KubeVirt Authors.
+Copyright 2019 The KubeVirt Authors.
```

Comment:
Good catch! Rather than remove 2018, this should read:

Copyright 2018, 2019 The KubeVirt Authors.

Reply:
The issue is, this is autogenerated. So if I change it, the travis will complain :)